### PR TITLE
added sirius video app

### LINKED ISSUE #5186: [CAL-244] sirius.video app
[https://sirius.video/sebastian](https://sirius.video/sebastian)

> From [Linear-GitHub Sync](https://synclinear.com)

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2022-10-24 at 23 30 43@2x

### CODE REVIEW

File: packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          <span className="mr-2">{isSunrise ? <Icon.FiSunrise /> : <Icon.FiSunset />}</span>I am an AppCard
+          for Event with Title: {eventType.title}
+        </div>{" "}
+        <div className="mt-2">
+          Edit <span className="italic">packages/app-store/{app.slug}/extensions/EventTypeAppCard.tsx</span>{" "}
+          to play with me
+        </div>
+      </div>
+    </AppCard>
+  );
```
Comment (PeerRich) [inline packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx:40]: @hariombalhara how do I remove this? in the new app store CLI you can only select:

- user
- eventType

but what about apps that don't need any settings?

Comment (hariombalhara) [reply packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx:40]: That would be the `user` app for now. User app doesn't generate EventTypeAppCard or GeneralSettings component.

Comment (PeerRich) [reply packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx:40]: ah how do I undo that?

Comment (hariombalhara) [reply packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx:40]: There is no way to change the type later using CLI. Let me quickly fix it by doing manual cleanup.

**Review by hariombalhara — APPROVED** [review]

File: packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx (hunk 1 above)
Comment (hariombalhara) [reply packages/app-store/sirius_video/extensions/EventTypeAppCard.tsx]: Done

**Pull request merged**